### ISSUE TITLE
[2.1.x] Volt: missing PHP closing tag for compiled macros

### ISSUE THREAD

**Issue by lajosbencz** [phalcon/cphalcon#10945]
Compiled macros are not closed properly:

Volt template:

```volt
{%- macro foobar() %}
    <div class="foobar">
        blabla
    </div>
{%- endmacro %}

<div class="container">
    {{ flashSession.output() }}
    <div class="content">
        {{ content() }}
    </div>
</div>
```

Compiled output:

```php
<?php $this->_macros['foobar'] = function() { ?>
    <div class="foobar">
        blabla
    </div><?php }; 

<div class="container">
    <?php echo $this->flashSession->output(); ?>
    <div class="content">
        <?php echo $this->getContent(); ?>
    </div>
</div>
```

```Parse error: syntax error, unexpected '<' in ....volt.php on line 6```

**Comment by lajosbencz** [comment]
Strange... according to the source https://github.com/phalcon/cphalcon/blob/880b4fde422c1e7a112d5546e274d12f6784442a/phalcon/mvc/view/engine/volt/compiler.zep#L1935 a closure binding should be generated before the closing tag, which is also missing...

I've cloned and used 95f665efe5cffbd31a1dd39c49c59d782b884d2c

**Comment by andresgutierrez** [comment]
Fixed in the 2.0.x branch
